ark/analysis/spatial_analysis.py

```diff
@@ -51,21 +50,20 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
 
     values = []
 
-    if excluded_colnames is None:
-        excluded_colnames = ["cell_size", "Background", "HH3",
-                             "summed_channel", "label", "area",
-                             "eccentricity", "major_axis_length", "minor_axis_length",
-                             "perimeter", "fov"]
-
     # Error Checking
-    if not np.isin(excluded_colnames, all_data.columns).all():
+    if not np.isin(excluded_channels, list(all_data.columns.values)).all():
```

Comment:
I think this will fail for None. Either way, we should add a test checking for when it's None